packages/clay-charts/src/ChartBase.js

```diff
@@ -103,6 +103,8 @@ const ChartBase = {
 			return;
 		}
 
+		this.cleanup();
```

Comment:
Do we really need to add this additional lifecycle method? Wouldn't it be enough to just implement `disposed()` in `DataComponent`

Reply:
Yes, you're right; that's the thing to do, thanks.